contributing.md

```diff
@@ -1,3 +1,9 @@
+### Follow Conventional Commits
+
+This repository enforces the
+[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+specification for for composing titles for pull requests as well as for issues.
```

Comment:
@not-ivy I accidentally made a typo in my above message. Please remove the 2nd "for"! 😂

Reply:
You're the one here who speaks English natively! xD Just make it sound more *natural*, you know?